en/docs/ipaas/ballerina/developing-integrations-with-ballerina.md

```diff
@@ -0,0 +1,42 @@
+# Developing Integrations with Ballerina
+
+WSO2 Ballerina is a programming language that is designed specifically for integration. Ballerina provides a 
+comprehensive set of features and tools for building and deploying integrations. It is a modern, cloud-native language 
+that offers a clean syntax and is designed to be simple, efficient, and expressive. Ballerina provides a range of 
+built-in libraries and tools for connecting to a variety of services and data sources, including APIs, databases, and 
+messaging systems. It also supports deployment on various platforms, including cloud and on-premises environments.
+
+WSO2 Ballerina-based integrations provide several advantages over traditional integration solutions. Firstly, Ballerina 
+provides a unified programming model for integrating different systems and services. This means that developers only 
+need to learn one language and one set of tools, making it easier to build and deploy integrations. Secondly, Ballerina 
+provides a range of security and performance features that make it easier to ensure the security and reliability of 
+integrations. Finally, Ballerina provides a cloud-native deployment model, making it easier to deploy and manage 
+integrations in the cloud.
+
+In summary, WSO2 Ballerina is a new programming language that is designed specifically for integration. It provides a 
+comprehensive set of features and tools for building and deploying integrations, and offers several advantages over 
+traditional integration solutions, including a unified programming model, security and performance features, and a 
+cloud-native deployment model.
+
+## Integration Types
+
+Choreo supports following types of MI Integrations
+
+- __Integration as an API__
```

Comment:
It is better to include a brief description of each type. How about the following?

- **Integration as an API**: These integrations exposed as APIs using HTTP.
- **Event-triggered Integration**: Integrations that are triggered by external events.
- **Scheduled Integration**: Integrations that are triggered automatically at scheduled time intervals
- **Manually-triggered Integration**:Integrations that you need to trigger manually.